src/Pipeline/LopezAcosta2019Tiling.jl

```diff
@@ -490,8 +492,12 @@ function get_final(
 
     #= opening to remove noise while preserving shape/size
     Note the different structuring elements for erosion and dilation =#
-    mask = sk_morphology.erosion(_img, se_erosion)
-    mask .= sk_morphology.dilation(mask, se_dilation)
+    mask = pyconvert(
+        Array, sk_morphology.erosion(Py(_img).to_numpy(), Py(se_erosion).to_numpy())
+    )
+    mask .= pyconvert(
+        Array, sk_morphology.dilation(Py(mask).to_numpy(), Py(se_dilation).to_numpy())
```

Comment:
Any ideas why we would need the sk_morphology erosion and dilation functions? One possibility would be if the sk_morphology functions are faster for unusual structuring elements.